src/chainparams.cpp

```diff
@@ -247,6 +251,7 @@ static Consensus::LLMQParams llmq400_85 = {
         .signingActiveQuorumCount = 4, // four days worth of LLMQs
 
         .keepOldConnections = 5,
+        .recoveryMembers = 100,
```

Comment:
Was the number of `.recoveryMembers` chosen arbitrarily? I noticed the smaller quorums use a higher percent of the overall LLMQ size.

Reply:
Yeah...mostly arbitrary. I chose the numbers in an absolute way because there wouldn't be a good percentage that would work well on all LLMQ types.